picard/ui/mainwindow/__init__.py

```diff
@@ -255,6 +255,20 @@ def setupUi(self):
         for function in ext_point_ui_init:
             function(self)
 
+        get_config().setting.setting_changed.connect(self.handle_settings_changed)
+
+    def handle_settings_changed(self, name, old_value, new_value):
+        if name == 'rename_files':
+            self.actions[MainAction.ENABLE_RENAMING].setChecked(new_value)
+        elif name == 'move_files':
+            self.actions[MainAction.ENABLE_MOVING].setChecked(new_value)
+        elif name == 'dont_write_tags':
+            self.actions[MainAction.ENABLE_TAG_SAVING].setChecked(not new_value)
+        elif name == 'clear_existing_tags':
+            self.metadata_box.update()
+        elif name in ('file_renaming_scripts', 'selected_file_naming_script_id'):
```

Comment:
I don't know if it makes any difference, but should this be a set rather than a tuple?

Reply:
In this context it doesn't make any difference though a set is more correct.

Reply:
Okay, thanks for clarifying.  You've caught me on that so many times in the past that I use sets whenever ordering is not required out of habit now.  😜